dir.props

```diff
@@ -274,6 +274,10 @@
 
   <!-- Setup properties per OSGroup -->
   <Choose>
+    <When Condition="'$(IsReferenceAssembly)' == 'true' OR'$(MSBuildProjectExtension)' == '.pkgproj'">
```

Comment:
I'm fine with the condition although I would expect reference assemblies to always be AnyOS.